src/pages/event.js

```diff
@@ -134,34 +109,59 @@ export function Event() {
       });
       setTrackedEvent(event.id);
     }
-  }, [event, succeededLoadingEvent, trackedEvent, setTrackedEvent]);
+  }, [event, trackedEvent, setTrackedEvent]);
 
-  useEffect(() => {
-    // Get new batch of tokens
-    if (eventId) {
-      dispatch(
-        fetchEventPageData({
-          eventId,
-          first: FETCH_POAPS_LIMIT,
-          skip: FETCH_POAPS_LIMIT * pageIndex,
+  function getNextBatch(batchSize) {
+    try {
+      return Promise.all(
```

Comment:
Without an `await`, this try/catch block will never work as just calling `Promise.all` will only generate a new promise that does not throw per-se.

Reply:
You are right, thanks!